modules/oadp-self-service-prerequisites.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-prerequisites_{context}"]
+= {oadp-short} Self-Service prerequisites
+
+Before you start using {oadp-short} Self-Service as a non-admin user, ensure you meet the following prerequisites:
+
+* The cluster administrator has configured the {oadp-short} `DataProtectionApplication` (DPA) CR to enable Self-Service. 
```

Comment:
We should include a DPA snippet here that shows the DPA configuration if it's not already handled in another section of this doc.

Reply:
Found it in section: Enabling OADP Self-Service